vector/src/main/res/layout/fragment_ftue_wait_for_email_verification.xml

```diff
@@ -0,0 +1,141 @@
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintHeight_percent="0.60"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Space
+        android:id="@+id/emailVerificationSpace1"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@id/emailVerificationLogo"
+        app:layout_constraintHeight_percent="0.10"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="spread_inside" />
+
+    <ImageView
+        android:id="@+id/emailVerificationLogo"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:adjustViewBounds="true"
+        android:background="@drawable/circle"
+        android:backgroundTint="?colorSecondary"
+        android:importantForAccessibility="no"
+        android:src="@drawable/ic_email"
+        app:layout_constraintBottom_toTopOf="@id/emailVerificationSpace2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHeight_percent="0.12"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/emailVerificationSpace1" />
+
+    <Space
+        android:id="@+id/emailVerificationSpace2"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@id/emailVerificationTitle"
+        app:layout_constraintHeight_percent="0.05"
+        app:layout_constraintTop_toBottomOf="@id/emailVerificationLogo" />
+
+    <TextView
+        android:id="@+id/emailVerificationTitle"
+        style="@style/Widget.Vector.TextView.Title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:transitionName="loginTitleTransition"
+        app:layout_constraintBottom_toTopOf="@id/emailVerificationSubtitle"
+        app:layout_constraintEnd_toEndOf="@id/ftueAuthGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/ftueAuthGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/emailVerificationSpace2"
+        tools:text="@string/ftue_auth_email_verification_title" />
+
+    <TextView
+        android:id="@+id/emailVerificationSubtitle"
+        style="@style/Widget.Vector.TextView.Subtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:gravity="center"
+        app:layout_constraintBottom_toTopOf="@id/emailVerificationSpace4"
+        app:layout_constraintEnd_toEndOf="@id/ftueAuthGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/ftueAuthGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/emailVerificationTitle"
+        tools:text="To confirm your email address, tap the button in the email we just sent to hello@world.org" />
```

Comment:
(presonal preference!) It's nice to have a real email in the preview, but I prefer to use the string which will be used at runtime (`ftue_auth_email_verification_subtitle` here), even if the preview will show `%s`. The reason if for find usage to find more usages.

Reply:
58a8a342b2123dd2a80547d78e4a7fb756f4f8ad